front/components/editor/input_bar/EmojiDropdown.tsx

```diff
@@ -48,7 +49,7 @@ const EMOJIS_MAP = new Map(emojis.map((emoji) => [emoji.name, emoji]));
 
 const POPULAR_EMOJIS = POPULAR_EMOJI_NAMES.map((name) =>
   EMOJIS_MAP.get(name)
-).filter((emoji) => emoji !== undefined);
+).filter((emoji): emoji is EmojiItem => emoji !== undefined);
```

Comment:
tsgo understands `POPULAR_EMOJIS` can't be `(T | undefined)[]` but not ts 5.4, so we need the type guard